library/components/TPdf.vue

```diff
@@ -18,9 +18,17 @@
         :class="is_loading && 'opacity-60'"
         @click="downloadPdf"
       >
-        <div class="flex items-center gap-1">
-          {{ is_loading ? "Generating PDF" : label }}
-          <t-loading-spinner v-if="is_loading" position="relative" />
+        <div
+          slot="handle"
```

Comment:
Do we need a handle slot here?

Reply:
Currently we don't but in future we can use this.